File: app/src/main/java/App/repositories/retrofit/service/UsuariosService.java
package App.repositories.retrofit.service;

import java.util.Map;

import App.repositories.retrofit.apiclient.RetrofitService;
import App.repositories.retrofit.responses.UsuariosResponse;
import App.repositories.room.entities.Usuarios;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.Headers;
import retrofit2.http.POST;

public interface UsuariosService {

    @Headers("Content-Type: application/json")
    @POST(RetrofitService.GET_USUARIOS)
    Call<UsuariosResponse> getUsuariosService(@Body Map<String, String> params);

    @Headers("Content-Type: application/json")
    @POST(RetrofitService.SET_USUARIOS_DELETE)
    Call<UsuariosResponse> deleteUsuariosService(@Body Map<String, String> params);

    @Headers("Content-Type: application/json")
    @POST(RetrofitService.SET_USUARIOS_UPDATE)
    Call<UsuariosResponse> updateUsuariosService(@Body Map<String, String> params);

    @Headers("Content-Type: application/json")
    @POST(RetrofitService.SET_USUARIO)
    Call<UsuariosResponse> setUsuarioService(@Body Map<String, String> params);

    @Headers("Content-Type: application/json")
    @POST(RetrofitService.GET_USUARIO)
    Call<Usuarios> getUsuarioService(@Body Map<String, String> params);

    @Headers("Content-Type: application/json")
    @POST(RetrofitService.GET_USUARIOEP)
    Call<Usuarios> getUsuarioEPService(@Body Map<String, String> params);
    
}

File: app/src/main/java/App/repositories/retrofit/service/UsuariosService.kt
package App.repositories.retrofit.service

import App.repositories.retrofit.apiclient.RetrofitService
import App.repositories.retrofit.responses.UsuariosResponse
import App.repositories.room.entities.Usuarios
import retrofit2.Call
import retrofit2.http.Body
import retrofit2.http.Headers
import retrofit2.http.POST

interface UsuariosService {
    @Headers("Content-Type: application/json")
    @POST(RetrofitService.GET_USUARIOS)
    fun getUsuariosService(@Body params: Map<String?, String?>?): Call<UsuariosResponse?>?

    @Headers("Content-Type: application/json")
    @POST(RetrofitService.SET_USUARIOS_DELETE)
    fun deleteUsuariosService(@Body params: Map<String?, String?>?): Call<UsuariosResponse?>?

    @Headers("Content-Type: application/json")
    @POST(RetrofitService.SET_USUARIOS_UPDATE)
    fun updateUsuariosService(@Body params: Map<String?, String?>?): Call<UsuariosResponse?>?

    @Headers("Content-Type: application/json")
    @POST(RetrofitService.SET_USUARIO)
    fun setUsuarioService(@Body params: Map<String?, String?>?): Call<UsuariosResponse?>?

    @Headers("Content-Type: application/json")
    @POST(RetrofitService.GET_USUARIO)
    fun getUsuarioService(@Body params: Map<String?, String?>?): Call<Usuarios?>?

    @Headers("Content-Type: application/json")
    @POST(RetrofitService.GET_USUARIOEP)
    fun getUsuarioEPService(@Body params: Map<String?, String?>?): Call<Usuarios?>?
}